### PR TITLE
ArrayExpression#value() method

### PR DESCRIPTION
**Pull request by Mike220889**
Tests added.
This branch depends on my previous pull request https://github.com/SBoudrias/AST-query/pull/17

See #16 and https://github.com/SBoudrias/gruntfile-editor/issues/26

### CODE REVIEW

**Comment by SBoudrias** [comment]
Hey, would you mind rebasing on master, it'll be easier to review thanks!

**Comment by Mike220889** [comment]
I've rebased on master. Thanks.

File: lib/nodes/ArrayExpression.js [hunk 1]
```diff
@@ -39,6 +40,21 @@ var ArrayExpression = module.exports = Base.extend({
...
+
+  /**
+   * Replace node with new value
+   * @param {String} value
+   * @return this
+   */
+  value: function (value) {
+    var val = valueFactory.create(value);
+    this.nodes.forEach(function (node) {
+      delete node.type;
```
Comment (SBoudrias) [inline lib/nodes/ArrayExpression.js]: There's issue with the indentation here

File: test/nodes/callexpression.js [hunk 2]
```diff
@@ -51,5 +51,26 @@ describe('CallExpression objects', function () {
...
+        this.tree = program('var foo = [1, 2, 3, 4];');
+      });
+
+      it('replaces itself with new value', function () {
+        var array = this.tree.var('foo').value();
+        var newArray = array.value('[8, 9]');
+        assert.equal(this.tree.var('foo').value().at(0).value(), 8);
+        assert.equal(this.tree.var('foo').value().at(1).value(), 9);
+        assert.throws(function(){
+			this.tree.var('foo').value().at(2).value();
```
Comment (SBoudrias) [inline test/nodes/callexpression.js]: indentation issue here

**Comment by SBoudrias** [comment]
Hey, sorry for the time it took to review, I had some more pressing matter in other project and could get by this one until today.

Some issues with the indentation, but once it's fixed and squashed in a single commit I'll merge.

**Comment by SBoudrias** [comment]
Hey @Mike220889, any chance you can follow up on the review? I'd really like to get this merged, thanks again!

**Comment by Mike220889** [comment]
Hi. Sorry, I'm on holiday at the moment so no laptop, I can review it in
about a weeks time.

**Comment by Mike220889** [comment]
@SBoudrias. Rebased, whitespace fixed and squashed into one commit. Thanks.

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**